src/zkevm_specs/evm/execution/return_revert.py

```diff
@@ -109,7 +121,8 @@ def return_revert(instruction: Instruction):
             rw_counter=Transition.delta(rwc_delta + 1),
             call_id=Transition.same(),
         )
-    else:
+
+    if not instruction.curr.is_root:
```

Comment:
this is just a style alignment with specs/circuit to use explicitly negate condition in previous `if` to replace `else` condition